core/src/main/resources/db/cgds.sql

```diff
@@ -185,14 +183,13 @@ CREATE TABLE `patient_list` (
 -- --------------------------------------------------------
 
 --
--- Table structure for table `patient_list_list`
+-- Table structure for table `sample_list_list`
 --
--- TODO rename the PATIENT_ID column, which is misleading
-CREATE TABLE `patient_list_list` (
+CREATE TABLE `sample_list_list` (
   `LIST_ID` int(11) NOT NULL,
-  `PATIENT_ID` int(11) NOT NULL,
-  PRIMARY KEY  (`LIST_ID`,`PATIENT_ID`),
-  FOREIGN KEY (`PATIENT_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
+  `SAMPLE_ID` int(11) NOT NULL,
+  PRIMARY KEY  (`LIST_ID`,`SAMPLE_ID`),
+  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
```

Comment:
missing foreign key to sample_list. Please add:
```sql
 FOREIGN KEY (`LIST_ID`) REFERENCES `sample_list` (`LIST_ID`)
```
I am not sure about the `ON DELETE CASCADE` part though...do we want this? I think that this could have some performance issues when removing large studies as a huge amount of deletes would have to be part of the same transaction (think of tables like `sample_cna_event`). @jjgao : what do you think about the `ON DELETE CASCADE` part?